app/views/policies_mailer/policy_update_review_closed.html.erb

```diff
@@ -0,0 +1,23 @@
+
+<% @title = t(".title") %>
+<% @sub_title = t(".subtitle", handle: @user.handle) %>
+
+<!-- Body -->
+<table width="100%" border="0" cellspacing="0" cellpadding="0" bgcolor="#ffffff">
+  <tr>
+    <td class="content-spacing" style="font-size:0pt; line-height:0pt; text-align:left" width="20"></td>
+    <td>
+      <table width="100%" border="0" cellspacing="0" cellpadding="0" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%"><tr><td height="35" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%">&nbsp;</td></tr></table>
+
+      <div class="h3-1-center" style="color:#1e1e1e; font-family:Georgia, serif; min-width:auto !important; font-size:20px; line-height:26px;">
+        <p>We're reaching out to inform you that the review period on our <a href="https://rubygems.org/policies/terms-of-service">Terms of Service</a>, <a href="https://rubygems.org/policies/privacy">Privacy Notice</a>, <a href="https://rubygems.org/policies/acceptable-use">Acceptable Use Policy</a>, and <a href="https://rubygems.org/policies/copyright">Copyright Policy</a> has closed. These policies are now in effect. You can read about the <a href="https://blog.rubygems.org/2025/03/20/introducing-new-policies.html">initial announcement of these policies here</a>.</p>
+        <br/>
+        <p>You will see a banner on RubyGems.org linking to the policies and asking you to acknowledge them. Once you acknowledge the policies, the banner will not reappear.</p>
```

Comment:
@colby-swandale I would prefer to add this once we decide after initial review. I'm afraid it will be left by accident in here.

Reply:
This reflects the current policy process. If this changes, we will include updating the email in the list of steps.